### PR TITLE
[ICDS Dashboard] Addition of fields in cas data export for pregnant and locatating women

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/0/1139198196866667/f

##### SUMMARY
https://docs.google.com/spreadsheets/d/1d4l5kMKXJGUjYXrJ8pbeS7ZN8jCHccWD10lCN1i5tbE/edit#gid=0

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/models/aggregate.py [hunk 1]
```diff
@@ -177,6 +177,21 @@ class CcsRecordMonthly(models.Model, AggregateMixin):
     anc_abnormalities = models.SmallIntegerField(blank=True, null=True)
     date_death = models.DateField(blank=True, null=True)
     person_case_id = models.TextField(blank=True, null=True)
+    husband_name = models.TextField(blank=True, null=True)
+    lmp = models.DateField(blank=True, null=True)
+    migration_status = models.PositiveSmallIntegerField(blank=True, null=True)
+    where_born = models.PositiveSmallIntegerField(
```
Comment (Rohit25negi) [inline custom/icds_reports/models/aggregate.py]: Migration for ccs_record monthly table will also happen yeah? @challabeehyv

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  left one comment.
Could you also add test cases to ensure the change is working as expected?

File: custom/icds_reports/models/aggregate.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/models/aggregate.py]: @Rohit25negi i see that css_record_monthly is not managed by migration. Do i add it in the 0050_move_to_django ?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  I think you missed adding fields to Cas data exports. 
These are the classes that serve data to three different CAS data exports https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/mbt.py#L51.

you can also reverse engineer this cas data export here https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/cas_export to find how they are working actually. 

Ping in case you want more clarity on that.

File: custom/icds_reports/tests/agg_tests/test_aggregation_script.py [hunk 2]
```diff
@@ -92,6 +92,8 @@ def _load_data_from_db(self, table_name, sort_key, filter_by=None):
     def _load_and_compare_data(self, table_name, path, sort_key=None, filter_by=None):
         # To speed up tests, we use a sort_key wherever possible
         #   to presort before comparing data
+        if 'css_record_monthly' in table_name:
```
Comment (Rohit25negi) [inline custom/icds_reports/tests/agg_tests/test_aggregation_script.py]: was this just for testing? Can you remove this if yes.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  test case failure looks related.

File: custom/icds_reports/tests/agg_tests/test_aggregation_script.py (hunk 2 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/test_aggregation_script.py]: aah yes

**Review by Rohit25negi — APPROVED** [review]
Looks good to me. Holding it off to merge until the data filling completes

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  can you update the migration file. i think this will conflict with master

File: custom/icds_reports/utils/aggregation_helpers/distributed/mbt.py [hunk 3]
```diff
@@ -145,7 +145,14 @@ def columns(self):
...
                 'valid_visits',
                 'dob',
-                'date_death')
+                'date_death',
+                'husband_name',
+                'lmp',
+                'migration_status',
+                'where_born',
+                'num_children_del',
+                'still_live_birth'
```
Comment (Rohit25negi) [inline custom/icds_reports/utils/aggregation_helpers/distributed/mbt.py]: 'name' is also needed here no? as the spec is saying.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
left two small comments. would be good to go after those are addressed.

**Review by Rohit25negi — APPROVED** [review]

**Comment by Rohit25negi** [comment]
@challabeehyv  could you resolve the merge conflict. We can merge after that?

**Comment by challabeehyv** [comment]
> @challabeehyv could you resolve the merge conflict. We can merge after that?

done.please merge once the tests are passed

**Review by Rohit25negi — APPROVED** [review]
Good to go once the test case passes.

**Comment by Rohit25negi** [comment]
@challabeehyv  test case failure looks related can you check.

**Pull request merged**